### PR TITLE
Fix queryCoord panic during query node down

### PR DESCRIPTION
**Pull request by SimFG**
issue: #21345
Signed-off-by: SimFG <bang.fu@zilliz.com>

### CODE REVIEW

File: internal/querycoordv2/balance/rowcount_based_balancer.go [hunk 1]
```diff
@@ -117,17 +119,21 @@ func (b *RowCountBasedBalancer) balanceReplica(replica *meta.Replica) ([]Segment
...
 		})
-		cnt := 0
-		for _, s := range segments {
-			cnt += int(s.GetNumOfRows())
-		}
-		nodesRowCnt[nid] = cnt
 
-		if nodeInfo := b.nodeManager.Get(nid); nodeInfo.IsStoppingState() {
+		if isStopping, err := b.nodeManager.IsStoppingNode(nid); err != nil {
+			log.Info("not existed node", zap.Int64("nid", nid), zap.Any("segments", segments))
```
Comment (jaime0815) [inline internal/querycoordv2/balance/rowcount_based_balancer.go:124]: log error information

File: internal/querynode/query_node.go [hunk 2]
```diff
@@ -328,7 +328,6 @@ func (node *QueryNode) Stop() error {
 		if err != nil {
 			log.Warn("session fail to go stopping state", zap.Error(err))
 		} else {
-			node.UpdateStateCode(commonpb.StateCode_Stopping)
```
Comment (jaime0815) [inline internal/querynode/query_node.go:331]: Why remove the stopping state here

Comment (SimFG) [reply internal/querynode/query_node.go:331]: it repeated.
image

File: internal/querycoordv2/balance/rowcount_based_balancer.go (hunk 1 above)
Comment (SimFG) [reply internal/querycoordv2/balance/rowcount_based_balancer.go:124]: Updated

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**